doc/usage/restructuredtext/domains.rst

```diff
@@ -367,6 +370,7 @@ are recognized and formatted nicely:
 * ``vartype``: Type of a variable.  Creates a link if possible.
 * ``returns``, ``return``: Description of the return value.
 * ``rtype``: Return type.  Creates a link if possible.
+* ``private``: Indicates the Python object is private member.
```

Comment:
We have to mention that private marker will not be shown to output.

Reply:
In my case, I divide the API modules into public and private, and only document the public APIs. Because of this, I would not use private markers to specify exclusions for documentation.
Also, I feel uncomfortable with the specification that it is not documented because it is private. `:undoc:` flag looks good for flags that are not documented.
<details><summary>In Japanese</summary>
私の場合、モジュールを公開用と非公開用に分けて、公開用だけをドキュメント化の対象とします。このため、私はドキュメント化の対象外を指定するためにprivateマーカーを使わないでしょう。
また私は、privateだからドキュメント化しないという仕様に違和感を感じます。ドキュメント化しないフラグなら `:undoc:` という名称がよさそうに思います。
</detail>

Reply:
Thank you for comment. -0 for `:undoc:`. I feel a bit strange to write an operation to info-field-list. It would be better to write its attribute or behavior. In addition, it is a bit surreal `:undoc:` marker is placed in docstring. It's already documented :-p  It might be a bit better to use `:hidden:` instead.

BTW, according to our policy, private members are not shown by default. They are only shown when `:undoc-members:` option is given. Therefore `:private:` is following that policy.

Reply:
I agreed that `:undoc:` is strange because it's a control command for sphinx behavior. However I think `:hidden:` is also unclear. I feel "what is hidden? hidden from what?"
So, I think name starts with `_` is best way to make it private (or separate private/public module).

Reply:
Agreed. withdrawing "hidden". Let's think another keyword.

>So, I think name starts with _ is best way to make it private (or separate private/public module).

Personally, I prefer not to use "_" prefix to represent private member. It's mere gentlemen's agreement, not an access control. So I'd like to provide ways to make members private. I know Zen of Python says "There should be one.", but...

Reply:
I and @shimizukawa discussed about this in face to face and agreed with "private" is enough good.

Addition to this, we also discussed the way to declaration. @shimizukawa worried about it is hard to understand `:private:` keyword is displayed to users or not. In this PR, I thought it is not shown to output document. But it is not obviously for writers of docstring. Now, other keywords are shown on document. So it is not easy to understand the behavior. Finally, we determined to use `:meta private:` instead to describe "private" is a metadata for the python object. We believe `:meta:` field will make info-field-list extendable in future like `:meta name: value`.

Reply:
Reimplemented with review comments!